Error with redirects while populating the database

The [catalog-update.yml](https://github.com/MobilityData/mobility-feed-api/blob/main/.github/workflows/catalog-update.yml) action is run when there is a change in the catalogs. 
After this PR: https://github.com/MobilityData/mobility-database-catalogs/pull/1238 we get this error:

```
INFO:GTFSDatabasePopulateHelper:Processing redirects
ERROR:GTFSDatabasePopulateHelper:
------ Failed to populate the database w***h sources.csv: 'float' object has no attribute 'split' -----

Traceback (most recent call last):
  File "/home/runner/work/mobility-feed-api/mobil***y-feed-api/api/src/scripts/populate_db.py", line 147, in initialize
    self.populate_db(session, fetch_url=fetch_url)
  File "/home/runner/work/mobility-feed-api/mobility-feed-api/api/src/scripts/populate_db_gtfs.py", line 269, in populate_db
    self.process_redirects(session)
  File "/home/runner/work/mobility-feed-api/mobility-feed-api/api/src/scripts/populate_db_gtfs.py", line 171, in process_redirects
    comments = raw_comments.split("|") if raw_comments is not None else []
               ^^^^^^^^^^^^^^^^^^
AttributeError: 'float' object has no attribute 'split'
Error: Process completed with exit code 1.
```
See https://github.com/MobilityData/mobility-feed-api/actions/runs/21372883655/job/61521758097

--- a/api/src/scripts/populate_db_gtfs.py
+++ b/api/src/scripts/populate_db_gtfs.py
@@ -168,7 +168,7 @@ class GTFSDatabasePopulateHelper(DatabasePopulateHelper):
                 continue
             feed = self.query_feed_by_stable_id(session, stable_id, None)
             raw_comments = row.get("redirect.comment", None)
-            comments = raw_comments.split("|") if raw_comments is not None else []
+            comments = str(raw_comments).split("|") if raw_comments is not None else []
             if len(redirects_ids) != len(comments) and len(comments) > 0:
                 self.logger.warning(f"Number of redirect ids and redirect comments differ for feed {stable_id}")
             for redirect_id in redirects_ids:

